### PR TITLE
Breakout geojson conversion methods

### PR DESCRIPTION
**Pull request by patrickarlt**
Use the new https://github.com/esri/arcgis-to-geojson for the ArcGIS <-> GeoJSON conversion.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,9 @@
     "John Gravois <jgravois@esri.com> (http://johngravois.com)"
   ],
   "dependencies": {
-    "leaflet": "^1.0.0-beta.2"
+    "arcgis-to-geojson-utils": "^1.0.0",
+    "leaflet-virtual-grid": "^1.0.0",
+    "leaflet": "1.0.0-beta.2"
```
Comment (jgravois) [inline package.json]: leaflet needs another `^`

File: package.json [hunk 2]
```diff
@@ -11,7 +11,9 @@
     "John Gravois <jgravois@esri.com> (http://johngravois.com)"
   ],
   "dependencies": {
-    "leaflet": "^1.0.0-beta.2"
+    "arcgis-to-geojson-utils": "^1.0.0",
+    "leaflet-virtual-grid": "^1.0.0",
```
Comment (jgravois) [inline package.json]: might as well bump to `1.0.1` while we're at it

**Comment by jgravois** [comment]
pulled this down and tested manually.  everything looks good.
i can fix my nitpicky stuff later.

**Comment by jgravois** [comment]
got tests running locally with Esri/arcgis-to-geojson#1
unless anyone disagrees, i'll hold off on merging this until we sort out why travis can't greenlight the new repo.

**Comment by patrickarlt** [comment]
@jgravois I'm going to merge this. We have to get someone on the GitHub Admin team to enable Travis.

**Pull request merged**